templates/vault/0/README.md

```diff
@@ -0,0 +1,32 @@
+# Vault #
+
+### Info:
+
+This template deploys a Hashicorp Vault server along with a Rancher LoadBalancer.  Once it is deployed, you will have a working Vault server ready to be scaled up to meet your environment's needs.
+
+The template is designed to be flexible in how you configure it; you can either statically bind Vault to an existing Consul stack, or ignore Consul all together and paste in your own Vault configuration.
+
+This catalog item uses these two main containers:
+* [Vault](https://www.vaultproject.io) - The official Hashicorp Vault image
+* [Rancher LoadBalancer](https://hub.docker.com/r/rancher/lb-service-haproxy/) - Rancher's own official HAProxy load balancer
+
+## Deployment:
+1. Select the catalog item and choose a version from the drop-down box
+2. Adjust any values on the page to meet your needs.
+3. Select a backend type and specify values specific to that type (i.e.: for Consul: "address":"SERVICE:8500","path":"myNewVault"
+4. Different ports to listen on (NOTE: Vault ALWAYS listens on port 8200, but you can adjust the LoadBalancer ports to any that are acceptable to your environment; the LoadBalancer handles routing between the port you specify and port 8200 (and 8201) in Vault itself.
+5. Specify the Volume Driver for pesistent mounting of Vault's FILE backing store, and CONFIGURATION
+6. Specify how many vault servers you want in your cluster (You'll always only get only 1 load balancer)
+6. Finally, once the stack is up, you can use your normal Vault process to init, unseal, and more.
+7. Enjoy!
+
+## Backend Configuration
+This field is for specifying your backend configuration values.  You enter them in a name=value pair format just as you would in a Vault configuration file; with each separate element being on its own line.  For example:
```

Comment:
Description is no correct, not format name=value, not each element in a line. Should be in json format. 
```
This field is for specifying your backend configuration values in json format, `"name": "value"`
```